File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int count = 1;
        for(int i = 0; i < nums.size(); i++) {
            for(int j = i + 1; j < nums.size(); j++) {
                if (nums[i] == nums[j]) {
                    count++;
                    return true;
                }
            }
        }
        return false;

    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hMap = {} 
        for i in range(len(nums)):
            difference = target - nums[i]
            if difference in hMap:
                return [hMap[difference], i]
            hMap[nums[i]] = i

        return -1